supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -129,9 +129,9 @@
 [discrete]
 [[data-warehouse]]
 ==== image:images/yes.png[yes] Data Warehouse (noun)
-*Description*: In Red Hat Virtualization, the Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
+*Description*: In Red Hat Virtualization, the _Manager_ includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
```

Comment:
I don't think Manager needs to be italicized here since it's not the term we're introducing

Reply:
:heavy_check_mark: